cordova-lib/package.json

```diff
@@ -42,7 +42,7 @@
     "underscore": "1.7.0",
     "unorm": "1.3.3",
     "valid-identifier": "0.0.1",
-    "xcode": "0.6.7"
+    "xcode": "https://github.com/ogoguel/node-xcode/tarball/17926117b388d08f05ec2db1e1ad8262f67a2a8d"
```

Comment:
You should create PR with your changes to [node-xcode](https://github.com/alunny/node-xcode) instead of using tarball dependency